ares-proxy/main.go

```
@@ -45,17 +51,25 @@ func handleChatCompletion(broker *Broker) http.HandlerFunc {
 			return
 		}
 
 		// Send the response back to the client
-		w.Header().Set("Content-Type", "application/json")
-		w.Write(response)
+		w.Header().Set("Content-Type", response.ContentType)
+		if response.ContentType == "text/event-stream" {
+			w.Header().Set("Cache-Control", "no-cache")
+			w.Header().Set("Connection", "keep-alive")
+		}
+		w.Write(response.Body)
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)